frontend/src/features/properties/selector/map/PropertyMapSelectorFormView.tsx

```diff
@@ -5,24 +5,33 @@ import {
 } from 'components/maps/providers/SelectedPropertyContext';
 import { Section } from 'features/mapSideBar/tabs/Section';
 import * as React from 'react';
+import { useEffect } from 'react';
 
 import MapClickMonitor from '../components/MapClickMonitor';
 import { IMapProperty } from '../models';
 import PropertyMapSelectorSubForm from './PropertyMapSelectorSubForm';
 
 export interface IPropertyMapSelectorFormViewProps {
   onSelectedProperty: (property: IMapProperty) => void;
+  initialSelectedProperty?: IMapProperty;
 }
 
 const PropertyMapSelectorFormView: React.FunctionComponent<IPropertyMapSelectorFormViewProps> = ({
   onSelectedProperty,
+  initialSelectedProperty,
 }) => {
   const { setCursor } = React.useContext(SelectedPropertyContext);
 
   const [selectedProperty, setSelectedProperty] = React.useState<IMapProperty | undefined>(
-    undefined,
+    initialSelectedProperty,
   );
 
+  useEffect(() => {
+    if (initialSelectedProperty !== undefined) {
+      setSelectedProperty(initialSelectedProperty);
+    }
+  }, [initialSelectedProperty]);
+
```

Comment:
question - why do we need this effect here? Wouldn't **selectedProperty** always take the value of **prop.initialSelectedProperty** ?